client/components/bulk-select/test/index.js

```diff
@@ -1,77 +1,78 @@
 /**
  * @jest-environment jsdom
  */
-
-import { assert } from 'chai';
-import { mount, shallow } from 'enzyme';
+import { fireEvent, render } from '@testing-library/react';
+import '@testing-library/jest-dom';
```

Comment:
Have we been importing this in other tests? Why do we need it for this one?

Reply:
Yes. It introduces a few [convenient matchers](https://github.com/testing-library/jest-dom#table-of-contents) to Jest's assertions. Namely here we're using the `toHaveClass`, but there are plenty other useful ones, like `toBeVisible`, `toHaveFocus` and so on.